docs/static/releasenotes.asciidoc

```diff
@@ -11,9 +11,12 @@ This section summarizes the changes in the following releases:
 [[logstash-5-6-3]]
 === Logstash 5.6.3 Release Notes
 
+* Known Issue: Running Logstash with log level set to debug will cause the pipeline to crash
+* Improved Pipeline performance
```

Comment:
@robbavey probably don't want to capitalize "Pipeline" here if we didn't also capitalize it in the above point?